DataFormats/Headers/src/DataHeader.cxx

```diff
@@ -27,14 +27,15 @@ const o2::Header::SerializationMethod o2::Header::gSerializationMethodFlatBuf("F
 //__________________________________________________________________________________________________
 //possible data origins
 const o2::Header::DataOrigin o2::Header::gDataOriginAny    ("***");
-const o2::Header::DataOrigin o2::Header::gDataOriginInvalid("   ");
+const o2::Header::DataOrigin o2::Header::gDataOriginInvalid("INV");
```

Comment:
then also for the longer ones: `FFFFFFF\0` etc... to be consistent with the invalid hex values of 0xFFFFFFFF...